rayon-core/src/scope/mod.rs

```diff
@@ -15,6 +15,8 @@ use unwind;
 #[cfg(test)]
 mod test;
 
+/// Represents a fork-join scope which can be used to spawn any number 
+/// of tasks. See [`Scope()`] for more information.
```

Comment:
@petezhut it's just markdown; you can check the output locally by running `cargo doc`, I think